package.json

```diff
@@ -24,8 +24,8 @@
     "vue2-ace-editor": "^0.0.15"
   },
   "peerDependencies": {
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react": "^17.0.2 || ^18.3.1",
+    "react-dom": "^17.0.2 || ^18.3.1"
```

Comment:
I believe in updating this; we need to change something more than that.

Reply:
@DwCleb Okay I'm currently working on this one. I will tag you once i was complete with the changes that i'm working on.